libtiny_tui/src/config.rs

```diff
@@ -14,6 +14,8 @@ pub use termbox_simple::*;
 pub(crate) struct Config {
     #[serde(default)]
     pub(crate) colors: Colors,
+    #[serde(default = "text_field_wrap_default")]
```

Comment:
I think default for `bool` is already `false` so you could simply use `#[serde(default)]` and remove `text_field_wrap_default` function.